vm/src/import.rs

```diff
@@ -22,8 +21,8 @@ fn import_module(vm: &mut VirtualMachine, module: &String) -> PyResult {
     }
 
     // Check for Rust-native modules
-    if let Some(module) = stdlib::get_modules(vm).get(module) {
-        return Ok(module.clone());
+    if let Some(module) = vm.stdlib_inits.get(module) {
```

Comment:
Maybe it is a good idea to implement some sort of function which `get_or_create`'s a module based on a name. This function then will check a cache `HashMap<String, PyObjectRef>` where the object ref is a module ref, and if not present, calls an init function on the module, so it is loaded lazily. On the other hand, we might preload all builtin modules, so that the initialization order is deterministic at all times.

Reply:
Isn't that cache just `sys.modules`?

(We don't currently put anything in there, see #109.)

Reply:
Ha! I totally forgot about that, you are right!